charts/simple-app/values.yaml

```diff
@@ -228,10 +260,10 @@ affinity: {}
 tests:
   connection:
     # -- The command used to trigger the test.
-    command: ['curl']
+    command: ['curl', '--retry-connrefused', '--retry', '5']
     # -- A list of arguments passed into the command. These are run through the tpl function.
     args:
-      - '{{ include "simple-app.fullname" . }}:{{ .Values.service.port }}'
+      - '{{ include "simple-app.fullname" . }}'
```

Comment:
this is reflecting the decision that your container runs on port 80, plaintext, & ssl termination happens in a sidecar.  corrrect?

Reply:
Yes - though more accurately, this is reflecting the _exact case described in the simple-app values.yaml file_.. which is a simple nginx service listening on port 80.